lib/modules/modules.go

```diff
@@ -147,10 +150,11 @@ func (p *defaultModules) PrintVersion() {
 // Features returns supported features
 func (p *defaultModules) Features() Features {
 	return Features{
-		Kubernetes: true,
-		DB:         true,
-		App:        true,
-		Desktop:    true,
+		Kubernetes:        true,
+		DB:                true,
+		App:               true,
+		Desktop:           true,
+		ModeratedSessions: false,
```

Comment:
Since this is different than other features, I'd add a comment explaining why.

```suggestion
		ModeratedSessions: false, // moderated sessions is supported in enterprise only
```